tests/detect-bytemath-mult-04/input.pcap

```diff
@@ -0,0 +1 @@
+../detect-bytemath-01/input.pcap
```

Comment:
This needs to be in test.yaml, right @inashivb ?

Reply:
Yes, something like:
```
pcap: ../base64-issue-5223/input.pcap
```
See file: https://github.com/OISF/suricata-verify/blob/master/tests/bug-5464-verdict-02/test.yaml#L4